File: 2st/Yeonwoo/왼쪽 오른쪽.py
def solution(str_list):
    answer = []
    for i in range(len(str_list)):
        if str_list[i]=='l':
            answer = str_list[:i]
            break
        elif str_list[i]=='r':
            answer = str_list[i+1:]
            break
    return answer

File: 3st/Juram/programmer/왼쪽 오른쪽.js
function solution(str_list) {
    let answer = [];
    for(let i = 0; i<str_list.length; i++){
        if(str_list[i] == "l") return str_list.slice(0,i);
        if(str_list[i] == "r") return str_list.slice(i+1); 
    }
    return [];
}